setup/Swix/Directory.Build.props

```diff
@@ -1,8 +1,9 @@
 <Project>
+
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
-  <Import Project="$(NuGetPackageRoot)\MicroBuild.Core\$(MicroBuildCorePackageVersion)\build\MicroBuild.Core.props" />
 
   <PropertyGroup>
+    <VisualStudioInsertionComponent>Microsoft.FSharp</VisualStudioInsertionComponent>
     <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
```

Comment:
><Configuration Condition="'$(Configuration)' == ''">Debug</Configuration> [](start = 4, length = 73)

Not needed.

Reply:
Why is this not needed?  I though this was how the .swixproj under this directory specify that they are part of `Microsoft.FSharp.vsman`?

Reply:
I meant the `<Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>` line. The `VisualStudioInsertionComponent` is indeed needed.